wiki/Mapping_Techniques/Formations/en.md

```diff
@@ -42,7 +42,7 @@ However, the difference here is that this forces the player to go along the edge
 
 Nominator: [mm201](https://osu.ppy.sh/users/30655)
 
-Honeycomb beats is a stream of hit circles that are usually half beats apart.
+**Honeycomb beats** is a stream of hit circles that are usually half beats apart.
```

Comment:
are streams* ? You use plural for some stuff so i don't know

Reply:
```suggestion
**Honeycomb beats** are streams of hit circles that are usually half beats apart.
```